### PR TITLE
Dev to Integration R2 "Code Freeze"

### PR DESCRIPTION
**Pull request by druth314**
This pull request is meant to finalize any features in the ETL process for R2. Some debugging may still be necessary

### CODE REVIEW

File: cdatransform/extract/idc.py [hunk 1]
```diff
@@ -101,88 +122,158 @@ def download_blob(self):
...
         query += """ARRAY_AGG(STRUCT("""
-        query += self.add_entity_fields('File')
+        query += self.add_entity_fields("File")
         query += """)) as File """
         # add WHERE statement if just looking for specific patients
         query += """FROM `""" + self.source_table + """`"""
         query += self.build_where_patients()
-        query += """ GROUP by id"""
+        query += """ GROUP by id, species, collection_id"""
+        print(query)
```
Comment (dionboles-asym) [inline cdatransform/extract/idc.py:226]: did you mean to keep this print statment

**Review by dionboles-asym — APPROVED** [review]
Good code from just reading it over i did not run it. Did you upload your requirements.txt?

File: cdatransform/extract/idc.py (hunk 1 above)
Comment (druth314) [reply cdatransform/extract/idc.py:226]: I could remove it if necessary. It just helped me figure out what to fix if the query goes wrong for some reason

Comment (dionboles-asym) [reply cdatransform/extract/idc.py:226]: I understand completely i just wanted to make sure it was not a mistake left in.

**Pull request merged**